JASP-Engine/JASP/R/correlationbayesianpairs.R

```diff
@@ -483,41 +483,21 @@ CorrelationBayesianPairs <- function(dataset=NULL, options, perform="run", callb
 					}
 					
 					errorMessage <- NULL
-					
+					errors <- .hasErrors(dataset, perform = "run", message = 'short', type = c('observations','variance', 'infinity'),
```

Comment:
The statement right above checks for 1 or -1 correlation, this could possibly be added as a built-in check of .hasErrors(). 
Is this sort of check used in other correlation code?